presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1469,6 +1469,32 @@ public void testCreateAndInsert()
         assertEqualsIgnoreOrder(actualAfterTransaction, expected);
     }
 
+    @Test
+    public void testRenamePartitionColumn()
+            throws Exception
+    {
+        for (HiveStorageFormat storageFormat : HiveStorageFormat.values()) {
+            testRenamePartitionColumn(storageFormat);
+        }
+    }
+
+    private void testRenamePartitionColumn(HiveStorageFormat storageFormat)
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_partitioned_table (" +
```

Comment:
Reusing table names inside this test would result in test failures when ran concurrently.